### PR TITLE
Development

### PR DESCRIPTION
**Pull request by serdg0**
PR for code review

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+                <button id="signin-button">
+                    <span class="whitelock"></span>
+                    Sign In
+                </button>
+            </div>
+        </div>
+        <header>
+            <img src="logos/main-logo.png" alt="Intuit">
+        </header>
+        <header class="logos-box">
```
Comment (Torres-ssf) [inline index.html]: You don't need to use 2 headers tags inside the same container. Use just one header tag and use 2 div to separate the main logo from the other logos.

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
Hi there, you did great work on this project 👏. But it needs 2 minor changes. I've highlighted one and the other one is:
1- You have to fill your readme file with a description of this project.
Make these 2 changes and submit for code review again.

**Review by bellom — CHANGES_REQUESTED** [review]
Hi @serdg0 

**Your next re-review should be :+1:** 
_**Kindly work on the  following;**_ 
1. According to the last review, you were told to update your README, follow the example below; 

image

2. It will be great to add lock icon to your web page similar to the original site, check for example below; 

image


3. Run your code on (https://validator.w3.org) and fix the errors.
[i ran your code on w3 validator, you have just one errors, kindly fix that.] 

**Free free to message me on slack if you need help.** @ sean bellom

Submit a new code review request. Thanks!

**Review by bellom — APPROVED** [review]
## Congratulation !! :+1:

**Pull request merged**